src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, Color color) {
+        this.radius = radius;
+        super.color = color;
+    }

```

Comment:
Don't use a setter inside the constructor to set a property that can be set directly. Since 'radius' is a private field of this class, you can assign it directly without using a setter.